config/jobs/ci-infra/ci-infra-periodics.yaml

```diff
@@ -273,7 +273,7 @@ periodics:
     description: Runs go tests for prow developments in ci-infra 
   spec:
     containers:
-    - image: golang:1.17.9
+    - image: golang:1.17.10
```

Comment:
We could use the `golang-test` image with a moving tag for ci-infra jobs as well. WDYT?
Then we wouldn't need to manually adapt this.

Reply:
good idea, done 😄